src/dbmanager/dbmanager_test.go

```diff
@@ -37,12 +38,12 @@ func TestRouteID(t *testing.T) {
 	db, err := dbOpen()
 	require.NoError(t, err)
 	dbmanager := NewDBManager(db)
-	id1, err := dbmanager.insertRoute(7, 32, 44, 1500, "2019-02-24 08:30:00")
+	id1, err := dbmanager.insertRoute(context.Background(), 7, 32, 44, 1500, "2019-02-24 08:30:00")
```

Comment:
Better to write struct with params, so you will always know, what your params  here mean